bundles/org.openhab.binding.dwdunwetter/src/main/resources/OH-INF/i18n/dwdunwetter_de.properties

```diff
@@ -11,7 +11,7 @@ thing-type.dwdunwetter.dwdwarnings.description = DWD Unwetterwarnungen für ein
 # thing types config
 
 thing-type.config.dwdunwetter.dwdwarnings.cellId.label = Cell-ID
-thing-type.config.dwdunwetter.dwdwarnings.cellId.description = ID der abzufragende Zelle. Siehe https\://www.dwd.de/DE/leistungen/opendata/help/warnungen/cap_warncellids_csv. Es kann auch mittels % eine Gesamtmenge abgefragt werden, z.B. 8111% alle Gemeinden die mit 8111 anfangen.
+thing-type.config.dwdunwetter.dwdwarnings.cellId.description = ID der abzufragende Zelle. Siehe https\\\://www.dwd.de/DE/leistungen/opendata/help/warnungen/cap_warncellids_csv, nur IDs die mit einer 8 (Ausnahme für Berlin\: 7) beginnen werden unterstützt. Es kann auch mittels % eine Gesamtmenge abgefragt werden, z.B. 8111% alle Gemeinden die mit 8111 anfangen.
```

Comment:
This should probably have been caught in Crowdin review, but won't this be formatted as `http\://www.dwd.de/...`? Name of csv seems also different from original ("_csv.csv"):
`thing-type.config.dwdunwetter.dwdwarnings.cellId.description = ID of the area to retrieve weather warnings. For a list of valid IDs look at https://www.dwd.de/DE/leistungen/opendata/help/warnungen/cap_warncellids_csv.csv, only IDs starting with an 8 (exception for Berlin: 7) are supported. With the % sign at the end it is possible to query multiple cells at once. For example with 8111% are cells retrieved that starts with 8111.`

Reply:
OK, found it in Crowdin, and was formatted wrongly as expected. I have corrected it, now need @cweitkamp to approve.

Reply:
The PR will be updated after the change is validated in Crowdin.

Reply:
Should we wait for it and take all other changes as hostage, or merge this now as it reflects the current state of translations, and let correction appear in another PR?